api/working-with-extensions/publishing-extension.md

```diff
@@ -336,13 +336,15 @@ To verify a publisher:
 
    ![Validation submitted](images/publishing-extension/validation-submitted.png)
 
-   Once your TXT record has been validated, the Marketplace team will review your request and grant verification within 5 business days.
+   Once your TXT record has been validated, the Marketplace team will review your request and let you know the result within 5 business days. The validation includes, but is not limited to: domain, website and extensions content eligibility, legitimacy, trust and positive reputation. New publishers will need a consistent track record demonstrating these for at least 6 months. 
 
-Once verified, you will also see the corresponding badge next to your publisher name in the Visual Studio Marketplace publisher management page:
+If validation is passed, you will see the corresponding badge next to your publisher name in the Visual Studio Marketplace publisher management page:
 
 ![Verified publisher manage](images/publishing-extension/verified-publisher-manage.png)
 
-> **Note**: Any changes to the publisher display name will revoke the verified badge.
+> **Notes**:
+> - Any changes to the publisher display name will revoke the verified badge.
+> - Any violations from the publisher will revoke the verified badge.
```

Comment:
@mariaghiondea should we clarify which violations we're referring to? Is this related to the abovementioned checks as part of the publisher verification?

Reply:
This is related to the statement above and those checks. I wanted to keep this generic, because it can be security, privacy, ToU.

Reply:
OK. I'll merge and get the site published.

Reply:
Thanks for the edit. Are you OK we merge?

Reply:
yes, please. Thank you!